GitExtensions.ruleset

```diff
@@ -62,6 +62,7 @@
     <Rule Id="SA1300" Action="None" />
     <Rule Id="SA1303" Action="None" />
     <Rule Id="SA1304" Action="None" />
+    <Rule Id="SA1305" Action="Warning" />
```

Comment:
💭 I'm typically not a fan of this rule (I'm the main reason why it's disabled by default). I find that in most projects, it produces a bunch of false positives and very few warnings that need to be corrected. GitExtensions appears to be different. I'm especially fascinated by the use of `a` as a prefix for so many things.